Makefile

```diff
@@ -1,27 +1,29 @@
-include build.conf
 FORCE_PULL := 0
 DEV_BUILD  := 0
+ARCH       := amd64
+
+include build.conf
+include build.conf.$(ARCH)
 
 
 installer: minimal
 	docker build -t $(IMAGE_NAME):$(VERSION) .
 
-bin/rancheros:
+bin/ros:
 	mkdir -p $(dir $@)
-	go build -tags netgo -installsuffix netgo -ldflags "-X github.com/rancher/os/config.VERSION=$(VERSION) -linkmode external -extldflags -static" -o $@
-	strip --strip-all $@
+	ARCH=$(ARCH) VERSION=$(VERSION) ./scripts/mk-ros.sh $@
 
+./ros: bin/ros
+ifeq "$(ARCH)" "amd64"
+	ln -sf bin/ros ./ros
+else
+	ARCH=amd64 VERSION=$(VERSION) ./scripts/mk-ros.sh $@
+endif
 
 pwd := $(shell pwd)
 include scripts/build-common
 
 
-assets/docker:
```

Comment:
We shouldn't remove this.  I want rancher/os to directly reference Docker.  It's a PITA to have to build a seperate `rancher/docker` image just to have a different Docker binary.